x/oracle/handler.go

```diff
@@ -22,41 +27,98 @@ func NewHandler(k Keeper) sdk.Handler {
 	}
 }
 
-// handleMsgPriceFeed handles a MsgPriceFeed
-func handleMsgPriceFeed(ctx sdk.Context, keeper Keeper, pfm MsgPriceFeed) sdk.Result {
+// handleMsgPricePrevote handles a MsgPricePrevote
+func handleMsgPricePrevote(ctx sdk.Context, keeper Keeper, ppm MsgPricePrevote) sdk.Result {
+	valset := keeper.valset
+
+	if !ppm.Feeder.Equals(ppm.Validator) {
```

Comment:
Can `ppm.Feeder` ever equal `ppm.Validator`? the former is of the type `sdk.AccAddress` and the second is of the type `sdk.ValAddress`

Reply:
This will compare Bytes of each variable. So I can equal for same byte addresses.